### ISSUE TITLE
Помогите не понимаю где запускать

### ISSUE THREAD

**Issue by Deni61** [Priler/jarvis#103]
Я не шарю за языки и не понимаю что запускать в jarvis v.0.0.3, что-бы он запустился!
Помогите пожалуйста!

**Comment by loop-lab** [comment]
Установи rust и npm, потом в /gui выполни npm install. В корне выполни cargo install tauri-cli и потом одну из команд из README

**Comment by SlenGuard** [comment]
все установил, тоже не шарю за языки, пожете пожалуйста пояснить что такое /gui.
также пробовал вести в cmd команду cargo tauri build , выдает ошыбку:
error: no such command: `tauri`

        Did you mean `miri`?

        View all installed commands with `cargo --list`
        
        помогите пожалуйста.(сорри за ошыбки, я не русский)

**Comment by loop-lab** [comment]
Ты не выполнил cargo install tauri в корне проекта. /gui - каталог проекта где лежит интерфейс

**Comment by SlenGuard** [comment]
cargo install tauri-cli
    Updating crates.io index
  Installing tauri-cli v1.4.0
    Updating crates.io index
   Compiling proc-macro2 v1.0.60
   Compiling quote v1.0.28
   Compiling unicode-ident v1.0.9
   Compiling cfg-if v1.0.0
   Compiling autocfg v1.1.0
   Compiling winapi v0.3.9
error: linker `link.exe` not found
  |
  = note: program not found

note: the msvc targets depend on the msvc linker but `link.exe` was not found

note: please ensure that Visual Studio 2017 or later, or Build Tools for Visual Studio were installed with the Visual C++ option.

note: VS Code is a different product, and is not sufficient.

error: could not compile `quote` (build script) due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `proc-macro2` (build script) due to previous error
error: could not compile `winapi` (build script) due to previous error
error: failed to compile `tauri-cli v1.4.0`, intermediate artifacts can be found at 


выдает ошыбку подскажыте что делать

**Comment by loop-lab** [comment]
А у вас рас установлен по иструкции для vs code?

**Comment by SlenGuard** [comment]
да

**Comment by loop-lab** [comment]
https://skr.sh/sKVrwDLDkga - Вы точно выполнили эти шаги пере установкой самого Rust?
https://doc.rust-lang.ru/book/ch01-01-installation.html

**Comment by Zaitun101** [comment]
Получилось в итоге запустить? Дошел до этапа cargo tauri build, и теперь получаю такую ошибку. Может кто-то сталкивался или знает как решить?

error: failed to run custom build command for `jarvis-app v0.0.3 (C:\Coding\jarvis\gui\src-tauri)`

Caused by:
  process didn't exit successfully: `C:\Coding\jarvis\gui\src-tauri\target\debug\build\jarvis-app-ce471d42f62dca44\build-script-build` (exit code: 1)
  --- stdout
  cargo:rerun-if-env-changed=TAURI_CONFIG
  cargo:rerun-if-changed=tauri.conf.json
  cargo:rustc-cfg=desktop
  cargo:rustc-cfg=dev
  path matching commands not found.
warning: build failed, waiting for other jobs to finish...

**Comment by SlenGuard** [comment]
нет, выполнил все шаги перед установкой самого Rust но все равно выдает такую же ошыбку

**Comment by Samuill** [comment]
даже чат гпт не знает в чем проблема 
  Compiling jarvis-app v0.0.3 (C:\Users\whyfl\Downloads\jarvis-master\jarvis-master\gui\src-tauri)
error: failed to run custom build command for `jarvis-app v0.0.3 (C:\Users\whyfl\Downloads\jarvis-master\jarvis-master\gui\src-tauri)`

Caused by:
  process didn't exit successfully: `C:\Users\whyfl\Downloads\jarvis-master\jarvis-master\gui\src-tauri\target\release\build\jarvis-app-0a52060d8a0f3aa1\build-script-build` (exit code: 1)
  --- stdout
  cargo:rerun-if-env-changed=TAURI_CONFIG
  cargo:rerun-if-changed=tauri.conf.json
  cargo:rustc-cfg=desktop
  path matching commands not found.
       Error failed to build app: failed to build app
       
       
       я скидівал ему tauri.conf.json и он говрит все верно не всеровно не билдится, не от имени админа, даже перезагрузка не помогла(ну тип а вдруг чет не встало), может кто-то может обьяснить? я не шарю раст

**Comment by GoodAlien** [comment]
Аналогичная ошибка, я так понимаю билд не рабочий...

error: failed to run custom build command for jarvis-app v0.0.3 (C:\Coding\jarvis\gui\src-tauri)
Caused by:
process didn't exit successfully: C:\Coding\jarvis\gui\src-tauri\target\debug\build\jarvis-app-ce471d42f62dca44\build-script-build (exit code: 1)
--- stdout
cargo:rerun-if-env-changed=TAURI_CONFIG
cargo:rerun-if-changed=tauri.conf.json
cargo:rustc-cfg=desktop
cargo:rustc-cfg=dev
path matching commands not found.

Может у кого-то осталась версия кода 0.02 ?

**Comment by daxumbra** [comment]
Добрый вечер. все сделал по инструкции так и не получилось запустить.
PS D:\Program Files\jarvis-master\jarvis-master\gui>  cargo tauri build
    Error `tauri.conf.json` error: "identifier" is a required property
    Error `tauri.conf.json` error on `build`: Additional properties are not allowed ('devPath', 'distDir', 'withGlobalTauri' were unexpected)
    Error `tauri.conf.json` error: Additional properties are not allowed ('package', 'tauri' were unexpected)
не подскажете что делать

**Comment by GiWeMX** [comment]
jarvis on  master 
❯ cargo tauri build
    Error `tauri.conf.json` error: "identifier" is a required property
    Error `tauri.conf.json` error on `build`: Additional properties are not allowed ('devPath', 'distDir', 'withGlobalTauri' were unexpected)
    Error `tauri.conf.json` error: Additional properties are not allowed ('package', 'tauri' were unexpected)
    
    вылетает такая ошибка чптишка подсказал поробовать cargo tauri init 
    не помогло

**Comment by amirhon1** [comment]
сначала надо в папке /app написать irm https://create.tauri.app/ps | iex выполнить условия и затем написать cargo tauri build 
можно еще cargo build и cargo d я все выполнил

**Comment by shall-11** [comment]
я вообще полный ноль в этой теме, может мне кто-нибудь написать что, куда и в какой последовательности делать
